docs/static/includes/module-features/bicepFeatures.csv

```diff
@@ -7,7 +7,7 @@
 6,res/app/container-app,✅,✅,✅,,,,✅
 7,res/app/job,✅,✅,✅,,,,✅
 8,res/app/managed-environment,✅,✅,✅,,,,✅
-9,res/automation/automation-account,✅,✅,✅,✅,✅,✅,✅
+9,res/automation/automation-account,✅,✅,✅,,,✅,
```

Comment:
@AlexanderSehr is this correct?

Reply:
No 😅 Already saw it. We need to fix the script. I presume it is not ready to detect `types/avm-common-types` references